packages/fetch/package.json

```
@@ -9,10 +9,10 @@
   "type": "module",
   "exports": {
     ".": {
-      "browser": "./src/lib.js",
       "require": "./dist/lib.node.cjs",
       "import": "./src/lib.node.js",
-      "types": "./dist/src/lib.node.d.ts"
+      "types": "./dist/src/lib.node.d.ts",
+      "browser": "./src/lib.js"
```

`types` should come first according to [Community Conditions Definitions](https://nodejs.org/api/packages.html#community-conditions-definitions) docs

https://nodejs.org/api/packages.html#community-conditions-definitions

```suggestion
      "types": "./dist/src/lib.node.d.ts",
      "require": "./dist/lib.node.cjs",
      "import": "./src/lib.node.js",
      "browser": "./src/lib.js"
```